src/components/SocialLogin/SocialLogin.js

```diff
@@ -1,7 +1,162 @@
-import React from "react";
+import React, { useRef, useState, useEffect } from 'react';
+import styled from 'styled-components';
 
-const SocialLogin = () => {
-  return <div>SocialLogin</div>;
+const SocialLogin = ({ loginIsOpen, setLoginIsOpen }) => {
```

Comment:
```suggestion
const SocialLogin = ({ isLoginOpen, setIsLoginOpen }) => {
```
작명 고민 잘 해주셨습니다
boolean type의 naming은 일반적으로 의문문 형태로 적습니다

Reply:
변경 완료 했습니당! 👍